deploy/clientcomm.tf

```diff
@@ -11,61 +11,77 @@ variable "ssh_public_key_path" {
   description = "The path to your SSH public key"
 }
 
+// TODO: This will probably have to come from the deployer's local environment.
 variable "twilio_account_sid" {
   description = ""
   default = "TODO ******TODO ******TODO *******"
 }
 
+// TODO: This will probably have to come from the deployer's local environment.
 variable "twilio_auth_token" {
   description = ""
   default = "TODO ******TODO ******TODO *******"
 }
 
+// TODO: This can probably be provisioned with terraform.
 variable "twilio_num" {
   description = ""
   default = "TODO ******TODO ******TODO *******"
 }
 
+// TODO: I think this is a constant, or at least derived from the hostname of
+// the deploy.
 variable "twilio_outbound_callback_url" {
```

Comment:
Yes, I believe you are correct on this TODO and the next; they are derived from the hostname of the deploy.